### PR TITLE
chore: update the volcano imagePullPolicy from IfNotPresent

### PR DESCRIPTION
**Pull request by Aju100**
Resolves: #2070

### CODE REVIEW

File: installer/volcano-development.yaml [hunk 1]
```diff
@@ -134,7 +134,7 @@ spec:
       priorityClassName: system-cluster-critical
       containers:
         - args:
-            - --tls-cert-file=/admission.local.config/certificates/tls.crt
+            - --tls-cert-file=imagePullPolicy/admission.local.config/certificates/tls.crt
```
Comment (Thor-wl) [inline installer/volcano-development.yaml:137]: Thanks for your contribution! Please update for the arm yaml files as well

**Comment by Aju100** [comment]
I have updated on arm file too.

**Review by shinytang6 — CHANGES_REQUESTED** [review]
Thx for your contributions.
But l think what you need to change are the files under `installer/helm/chart/*` and then use `make update-development-yaml` to generate `volcano-development-*` files automatically.

**Comment by Aju100** [comment]
@shinytang6 Resolves the automated generated yaml file for policy one

**Review by shinytang6 — APPROVED** [review]
/approve

**Comment by Thor-wl** [comment]
@Aju100 Hi. Are there some updates again? If not, I'll close this PR later.

**Pull request closed without merging**